### PR TITLE
rewrite "print" in python 3.X compatible way

### PR DESCRIPTION
**Pull request by chipitsine**

### CODE REVIEW

**Comment by cron2** [comment]
@mattock - merge or close?

**Comment by mattock** [comment]
@chipitsine could you resolve the merge conflict so that we can merge this PR?

**Comment by mattock** [comment]
I'll test this and resolve the merge conflict when I build the official installers for tap-windows6-9.23.1, i.e. tomorrow or on Wednesday. If I don't encounter any issues I'll merge this PR then.

File: buildtap.py [hunk 1]
```diff
@@ -1,5 +1,7 @@
 # build TAP-Windows NDIS 6.0 driver
 
+from __future__ import print_function
```
[the remark this answers is not in the document]
Comment (schwabe) [reply buildtap.py]: You only need those for python older than 2.7 and I don't think we care a bit about those

**Comment by chipitsine** [comment]
@mattock , I have resolved merge conflict

**Comment by mattock** [comment]
Looks good now, thanks!

**Pull request merged**